docs/installation_guide/upgrades/0_10_0.rst

```diff
@@ -0,0 +1,85 @@
+==============
+Version 0.10.0
```

Comment:
@SirOibaf half of the document refers to `0.10.0` and the other half to `0.10`. We can stick to `0.10` since there will not be breaking changes in the `0.10.x` series